doc/user/content/demos/business-intelligence.md

```diff
@@ -118,29 +118,18 @@ Metabase](/images/demos/bi_architecture_diagram.png)
     git clone git@github.com:MaterializeInc/materialize.git
     ```
 
-1. Move to the `demo/chbench` dir:
-
-    ```shell
-    cd <path to materialize>/demo/chbench
-    ```
-
-    You can also find the demo's code on
-    [GitHub](https://github.com/MaterializeInc/materialize/tree/{{< version >}}/demo/chbench).
-
 1. Deploy and start all of the components we've listed above.
 
     Note that pulling down all of the Docker images necessary for the demo takes
-    quite a bit of time (upwards of 10 minutes, even on very fast connections).
+    quite a bit of time (up to 10 minutes, even on very fast connections).
 
     ```shell
-    # Deploy MySQL, Debezium, Kafka et. al., Metabase, and Materialize.
-    # Initialize MySQL with some chBench data.
-    ./dc.sh up :init:
-    ./dc.sh up :demo:
-    # Run the chBench workload generator.
-    ./dc.sh demo-load
+    bin/mzconduct run chbench --workflow demo
```

Comment:
The command line doesn't work for me with the latest code from `master`:

``` bash-session
[cdo@ubuntu:materialize]$ bin/mzconduct run chbench --workflow demo
C> Executing chbench -> demo
C> ERROR: Unknown workflow: 'demo'. Expected one of: ci, demo-load, load-test, bring-up-source-data
```

Looks like it's `demo-load`?

Reply:
This PR also changes the name of that workflow to demo.

Reply:
I've added back a `demo-load` workflow that just delegates to the `demo` workflow, I'm not sure if we should prefer to not update the docs page to the new command until the next release of materialize for some reason @sploiselle @benesch?

Reply:
Yeah, we don't have a good story about backwards incompatible changes like this at the moment. I don't really know what to tell you! Both options are bad! If you wait, you'll probably forget. (At least if I were in your shoes I would.) If you do it now, the docs will be a bit wrong until the next release. Maybe @sploiselle has a magic bullet.

Reply:
ah so maybe leave the docs using the old thing, and then eventually point to the new thing, when we remember. To be clear it's not a big deal for them to have both, I just think `demo-load` is kind of the wrong word.

Reply:
Which I have now done.